src/client/entrypoints/main/board_tools.tsx

```diff
@@ -0,0 +1,125 @@
+import {BoardUpdateForm, NewBoardForm} from '_client/board_tools/board_form';
+import {BoardSelector, idSelector} from '_client/board_tools/board_selector';
+import * as UiUtil from '_client/common/ui_util';
+import {GameBoard} from '_client/game_board/controller/game_board';
+import {BoardModel} from '_client/game_board/model/board_model';
+import {BoardClient} from '_client/game_board/remote/board_client';
+import {DropdownSelector} from '_client/common/ui_components/dropdown';
+
+import React from 'react';
+import {loadBoard} from '_client/entrypoints/main/game_board';
+import ReactDOM from 'react-dom';
+import {Hideable} from '_client/common/ui_components/hideable';
+
+const NEW_BOARD_BUTTON = 'createNewBoard';
+const BOARD_FORM_STUB = 'createNewBoardFormStub';
+
+const MAIN_BOARD_STUB = 'mainBoard';
+
+const ACTIVE_SELECTOR_STUB = 'activeSelectorStub';
+const EDIT_SELECTOR_STUB = 'editSelectorStub';
+const EDITING_AREA_STUB = 'editingAreaStub';
+
+const SIDE_PANEL = 'sidePanelContent';
+
+const boardEditRow: React.CSSProperties = {
+  display: 'flex',
+  flexDirection: 'row',
+};
+
+export class EditingArea extends React.Component {
```

Comment:
This should be a functional component, because there are no class features that you are using. https://reactjs.org/docs/components-and-props.html#function-and-class-components

Reply:
Done!